packages/commcare/src/Adaptor.js

```diff
@@ -56,6 +56,7 @@ export function execute(...operations) {
 
 /**
  * Make a GET request to any commcare endpoint. The returned objects will be written to state.data.
+ * Adding an offset param will trigger an automatic pagination and the returned data is combined and written into state.data
```

Comment:
I'm afraid these docs are incorrect. Check the spec in the issue - is this the correct behaviour?